docs/DXIL.rst

```diff
@@ -267,6 +267,17 @@ The attribute 'waveops-include-helper-lanes' is utilized to indicate that wave o
 
 'fp32-denorm-mode' is employed to define the denorm mode for the function. The possible values for this attribute can be 'any', 'preserve', or 'ftz'.
 
+Module Flags
+------------
+
+DXIL uses `Module Flags Metadata <LangRef.html#module-flags-metadata>`_ to convey additional information about the module.
+
+The only module flags allowed in DXIL are "Dwarf Version" and "Debug Info Version".
```

Comment:
> The only module flags allowed in DXIL are ...

Implied in this is that any other module flags will need to be stripped out when lowering to DXIL.  May be worth calling out?

Reply:
Updated.